### PR TITLE
scikit-learn import and comment fix

### PR DESCRIPTION
**Pull request by underdogliu**
This is related with issue #1484

I found in our diarization-related scripts and runners, we ask the users to install `sklearn`, instead of `scikit-learn`. `sklearn`, on pip, as pointed out in the issue, is a void package while `scikit-learn` is the library we want. Of course, in the python scripts, it is still imported as `import sklearn`.

Tests have been done via pip and conda. For conda actually, we can't even find `sklearn`.

Note: seems like this PR contains some un-necessary changes due to my [black](https://github.com/psf/black) accent plugin in my vscode....if necessary I can check them out.

### CODE REVIEW

**Comment by anautsch** [comment]
Thank you @underdogliu :D

Please run linters & update this one too (hope I didn't miss it)
https://github.com/speechbrain/speechbrain/blob/1b5811cf1c60654723ff0f0b8b112ee832a6cdf3/recipes/UrbanSound8k/SoundClassification/train.py#L29

**Review by nauman-daw — CHANGES_REQUESTED** [review]
Thanks @underdogliu  
Kindly make the necessary changes to pass all the tests. They are mostly minor syntax updates. 
These changes looks good to me.
 @mravanelli   this PR can be merged once all the tests are passed.

**Comment by underdogliu** [comment]
@mravanelli Some of my changes on voxceleb2 results today are merged with this PR, as we discussed. Just for your information. I would not open another PR.

File: recipes/VoxCeleb/SpeakerRec/hparams/verification_ecapa.yaml [hunk 1]
```diff
@@ -8,8 +8,9 @@ seed: 1234
 __set_seed: !apply:torch.manual_seed [!ref <seed>]
 
 # Folders and train_log file
-data_folder: !PLACEHOLDER #e.g., /localscratch/voxceleb1
-output_folder: results/voxceleb1_2/speaker_verification_ecapa_big
+# data_folder: !PLACEHOLDER #e.g., /localscratch/voxceleb1
+data_folder: /data/xuechen/corpus/voxceleb1
```
Comment (anautsch) [inline recipes/VoxCeleb/SpeakerRec/hparams/verification_ecapa.yaml:12]: placeholder :)

File: recipes/VoxCeleb/SpeakerRec/hparams/verification_ecapa.yaml [hunk 2]
```diff
@@ -25,15 +26,16 @@ verification_file: https://www.robots.ox.ac.uk/~vgg/data/voxceleb/meta/veri_test
 # is downloaded from the speechbrain HuggingFace repository.
 # However, a local path pointing to a directory containing your checkpoints may also be specified
 # instead (see pretrainer below)
-pretrain_path: speechbrain/spkrec-ecapa-voxceleb
+# pretrain_path: speechbrain/spkrec-ecapa-voxceleb
+pretrain_path: results/ecapa_augment/1986/save/CKPT+2022-08-23+14-22-41+00
```
Comment (anautsch) [inline recipes/VoxCeleb/SpeakerRec/hparams/verification_ecapa.yaml:30]: please use the earlier one

File: recipes/VoxCeleb/SpeakerRec/hparams/verification_ecapa.yaml [hunk 3]
```diff
@@ -25,15 +26,16 @@ verification_file: https://www.robots.ox.ac.uk/~vgg/data/voxceleb/meta/veri_test
...
 
 # csv files
 train_data: !ref <save_folder>/train.csv
 enrol_data: !ref <save_folder>/enrol.csv
 test_data: !ref <save_folder>/test.csv
 
 batch_size: 8
-score_norm: s-norm  # z-norm t-norm s-norm (uncomment to enable it)
+# score_norm: s-norm  # z-norm t-norm s-norm (uncomment to enable it)
```
Comment (anautsch) [inline recipes/VoxCeleb/SpeakerRec/hparams/verification_ecapa.yaml:38]: to be discussed

**Comment by Adel-Moumen** [comment]
Hello, 

Is there any news on this PR, please? 😃

**Comment by mravanelli** [comment]
Hi @underdogliu, could you pelease fix the linters and the faling tests?

**Comment by underdogliu** [comment]
@mravanelli @anautsch all tests have been passed.

**Review by mravanelli — APPROVED** [review]

**Comment by mravanelli** [comment]
Thank you @underdogliu!

**Pull request merged**